mapping.5x.json

```diff
@@ -0,0 +1,112 @@
+{
+  "settings" : {
+    "number_of_shards" : 2,
+    "number_of_replicas" : 1,
+    "index": {
+      "analysis": {
+        "analyzer": {
+          "myCustomAnalyzer": {
+            "type": "custom",
+            "tokenizer": "myCustomTokenizer",
+            "filter": ["myCustomFilter1", "myCustomFilter2"],
+            "char_filter": ["myCustomCharFilter"]
+          }
+        },
+        "tokenizer": {
+          "myCustomTokenizer": {
+            "type": "letter"
+          },
+          "myCustomNGramTokenizer": {
+            "type" : "ngram",
+            "min_gram" : 2,
+            "max_gram" : 3
+          }
+
+        },
+        "filter": {
+          "myCustomFilter1": {
+            "type": "lowercase"
+          },
+          "myCustomFilter2": {
+            "type": "kstem"
+          }
+        },
+        "char_filter": {
+          "myCustomCharFilter": {
+            "type": "mapping",
+            "mappings": ["ph=>f", " u => you ", "ES=>Elasticsearch"]
+          }
+        }
+      }
+    }
+  },
+  "mappings" : {
+    "group" : {
+      "_source" : {
+        "enabled" : true
+      },
+      "_all" : {
+        "enabled" : true
+      },
+      "properties" : {
+        "organizer" : { "type" : "text" },
+        "name" : { "type" : "text" },
+        "description" : {
+          "type" : "text",
+          "term_vector": "with_positions_offsets"
+        },
+        "created_on" : {
+          "type" : "date",
+          "format" : "yyyy-MM-dd"
+        },
+        "tags" : {
+          "type" : "text",
+          "index" : "analyzed",
+          "fields": {
+            "verbatim" : {
+              "type" : "text",
+              "index" : "not_analyzed"
+            }
+          }
+        },
+        "members" : { "type" : "text" },
+        "location_group" : { "type" : "text" }
+      }
+    },
+    "event" : {
+      "_source" : {
+        "enabled" : true
+      },
+      "_all" : {
+        "enabled" : false
+      },
+      "_parent" : {
+        "type" : "group"
+      },
+      "properties" : {
+        "host" : { "type" : "text" },
+        "title" : { "type" : "text" },
+        "description" : {
+          "type" : "text",
+          "term_vector": "with_positions_offsets"
+        },
+        "attendees" : { "type" : "text" },
```

Comment:
On page 199 of the book (listing 7.10), there is an example of doing an aggregation on the `attendees` field, but it fails because there is no un-inverted index (`not_analyzed`).  So I changed the mappings to this:

```json
        "attendees" : {
          "type" : "text",
          "fields": {
            "keyword": {
              "type": "keyword",
              "ignore_above": 256
            }
          }
        },
```